crates/pg_configuration/src/database.rs

```diff
@@ -26,6 +26,10 @@ pub struct DatabaseConfiguration {
     /// The name of the database.
     #[partial(bpaf(long("database")))]
     pub database: String,
+
+    /// The connection timeout in seconds.
+    #[partial(bpaf(long("conn_timeout")))]
+    pub conn_timeout_secs: Option<u16>,
```

Comment:
Not sure, but maybe we can set the default via bpaf?

Reply:
If not, we should add it to the comments.

Reply:
I played around with using a bpaf fallback, but the `partial` macro requires that the fallback is wrapped in an Option.

If we want to use `display_fallback`, we would also need to implement Display for `Option<u16>`, but we can't because of the Orphan rule.

A quick fix is to add the fallback to the comment, wdyt?

Reply:
Ah, we can use the `debug` representation. Works for me, too.

<img width="538" alt="Screenshot 2025-02-09 at 18 27 12" src="https://github.com/user-attachments/assets/2e782231-29f7-4bbf-884b-797adec64f91" />